ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -202,3 +202,8 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-metal-ipi-nightly
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e
+  cron: "0 23 * * *"
```

Comment:
Could we change this to be more frequent? Say, once every 90 minutes?

See: https://github.com/openshift/release/pull/7869#issuecomment-602975009

Reply:
90 min can not be done in one crontab line. Can we go with 1 or 2 hours instead? I would also change job name from nightly to hourly (or just periodic) in this case.